### PR TITLE
 user should be able to initialize vault as onebox from the new UI

### PR DESCRIPTION
**Pull request by priyakirankumar**
https://github.com/Shippable/admiral/issues/1558

### CODE REVIEW

File: static/scripts/dashboard/dashboard2Ctrl.js [hunk 1]
```diff
@@ -1145,12 +1145,17 @@
       //console.log('Modal Back to old values ' , $scope.vm.initializeForm[fromCopy]);
     }
 
-    function showInstallLocationModal(sectionName) {
+    function showInstallLocationModal(sectionName, clickType) {
       var old = sectionName + '_old';
+      //console.log('get values', $scope.vm.initializeForm[sectionName]);
```
Comment (mohit5it9) [inline static/scripts/dashboard/dashboard2Ctrl.js]: remove the console line

File: static/scripts/dashboard/dashboard2.html [hunk 2]
```diff
@@ -216,45 +216,47 @@ <h3 class="portlet-title text-dark">
...
-                  <h5><code>Existing node details are updated.</code></h5>
-                </div>
+              <div class="col-md-4 alignTextRight">
+                <button class="btn btn-sm btn-default" ng-click="vm.showConfigModal('secrets')" ng-disabled="!vm.systemSettings.db.isInitialized">config</button>
+                <button class="btn btn-sm btn-default" ng-click="vm.showLogModal('secrets')" ng-disabled="!vm.systemSettings.secrets.isProcessing && !vm.systemSettings.secrets.isFailed && !vm.systemSettings.secrets.isInitialized">logs</button>
+              </div>
+            </div><!--row ends-->
+
+            <div class="row">
+              <div class="col-md-3">
```
Comment (mohit5it9) [inline static/scripts/dashboard/dashboard2.html]: should this be col-md-4?

File: static/scripts/dashboard/dashboard2Ctrl.js [hunk 3]
```diff
@@ -1145,12 +1145,17 @@
...
 
       if (sectionName === 'secrets') {
         $scope.vm.initializeForm[old] =  angular.copy($scope.vm.initializeForm.secrets);
-        $('#secrets-location-modal').modal('show');
+        if (clickType === 'dropdown' && $scope.vm.initializeForm.secrets.initType === 'admiral') {
```
Comment (mohit5it9) [inline static/scripts/dashboard/dashboard2Ctrl.js]: if $scope.vm.initializeForm.secrets.initType !== 'admiral'
we can get rid of the else then

Comment (priyakirankumar) [reply static/scripts/dashboard/dashboard2Ctrl.js]: We are opening the modal in the else portion. If it is == admiral, then we do not need the add node popup.

File: static/scripts/dashboard/dashboard2Ctrl.js (hunk 1 above)
Comment (priyakirankumar) [reply static/scripts/dashboard/dashboard2Ctrl.js]: will do. Thanks.

File: static/scripts/dashboard/dashboard2Ctrl.js (hunk 3 above)
Comment (priyakirankumar) [reply static/scripts/dashboard/dashboard2Ctrl.js]: Need to open the modal either from the dropdown or from the button. Both are required for easier usability. So open modal only if New/Existing is pressed. Or open if the button is clicked directly.

Comment (mohit5it9) [reply static/scripts/dashboard/dashboard2Ctrl.js]: ok

**Pull request merged**